kolibri/plugins/coach/assets/src/csv/fields.js

```diff
@@ -62,7 +62,7 @@ function padTime(interval) {
 export function avgScore(quiz = false) {
   return [
     {
-      name: coachStrings.$tr(quiz ? 'avgQuizScoreLabel' : 'avgScoreLabel'),
```

Comment:
@jredrejo No errors found in my testing, could you confirm that this edit is kosher? Thank you!

Reply:
The point was to unify `Average quiz score` and `Average score` (see below): they were used in several places interchangeably, and only quizzes can have scores, so the word `quiz` was redundant. 

Table column header in printed reports general use just the word `Score`, so no formatting should be affected by this change.